pkg/network/pasta/pasta.go

```diff
@@ -147,21 +141,18 @@ func (d *parentDriver) ConfigureNetwork(childPID int, stateDir, detachedNetNSPat
 	// `Couldn't open user namespace /proc/51813/ns/user: Permission denied`
 	// Possibly related to AppArmor.
 	cmd := exec.Command(d.binary, opts...)
-	cmd.Stdout = d.logWriter
-	cmd.Stderr = d.logWriter
```

Comment:
Where is the stderr now ?

Reply:
It's all in `out` -- that's what `out, err := cmd.CombinedOutput()` does.

Reply:
https://pkg.go.dev/os/exec#Cmd.CombinedOutput